src/index.html

```diff
@@ -17,11 +17,375 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
```

Comment:
The image source '../src/images/logo.png' might not be correct. Ensure the path is correct relative to the HTML file location.